src/components/gds-media-card/_index.scss

```diff
@@ -25,5 +13,5 @@
 }
 
 @mixin tags {
-  margin-top: 20px;
+  margin: 20px 8px 32px 8px;
```

Comment:
I guess we don't have sass variables for these yet? Should we?

Reply:
No, we don't have anything yet for spacing. We should think about what spacing is implemented through variables and what are intrinsic parts of the design.